modules/ClinGen.py

```diff
@@ -89,165 +47,78 @@ def process_gene_validity_curations(self, in_filename, out_filename, unmapped_di
 
     def generate_evidence_strings(self, filename):
 
-        # When reading csv file first extract date from second row and the skip header lines that don't contain column names
-        # CLINGEN GENE VALIDITY CURATIONS
-        # FILE CREATED: 2020-07-16
-        # WEBPAGE: https://search.clinicalgenome.org/kb/gene-validity
-        # +++++++++++,++++++++++++++,+++++++++++++,++++++++++++++++++,+++++++++,+++++++++,++++++++++++++,+++++++++++++,+++++++++++++++++++
-        # GENE SYMBOL,GENE ID (HGNC),DISEASE LABEL,DISEASE ID (MONDO),MOI,SOP,CLASSIFICATION,ONLINE REPORT,CLASSIFICATION DATE
-        # +++++++++++,++++++++++++++,+++++++++++++,++++++++++++++++++,+++++++++,+++++++++,++++++++++++++,+++++++++++++,+++++++++++++++++++
-
-        # Read first two rows of file to extract date
-        with open(filename) as f:
-            f.readline() # Don't do anything with first row
-            second_row = f.readline()
-            file_created_date = second_row.split(": ")[1].strip() # Extract date and clean it
-
-
+        # When reading csv file skip header lines that don't contain column names
         gene_validity_curation_df = pd.read_csv(filename, skiprows= [0,1,2,3,5], quotechar='"')
         for index, row in gene_validity_curation_df.iterrows():
             print("{} - {}".format(row["GENE SYMBOL"], row["DISEASE LABEL"]))
```

Comment:
Do we want to keep print statements? Wouldn't make more sense to have log statements instead? We can also do an other round of iteration when when we synchronise all logging, and print.

Reply:
You're right, that's a mistake - I'll fix it.

Reply:
You're right, that's a mistake - I'll fix it.